### ISSUE TITLE
Curate resource links for API reference

### ISSUE THREAD

**Issue by shmoradims** [dotnet/machinelearning#2356]
There are many topics that repeat throughout the API reference. For each topic, we want to have a separate link that contains more details, instead of duplicating content, or worse, stating the obvious like the following:
* Bias: The predictor's bias term.
* L2Weight: L2 regularization weight for trainer

Table below captures all the topics that we want to cover. These are the topics that apply to many APIs. For example, we have 4 or 5 cross validation extension methods, one per ML task. We can have one page explaining CV, instead of duplicating content. 

Some topics already exist in [glossary](https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/glossary). We need to have a discussion about our linking strategy later. For now let's keep the list here. I suggest using the LinkPlaceholder string (tmpurl_*) in the XMLs, so that we can easily find-replace with the final link.

| Topic | Description | In Glossary? | Link Placeholder | Link |
| ------------- |-------------| -----| -----| -----|
| Regularization | L1, L2, and other topics related to regularization | no | tmpurl_regularization | https://en.wikipedia.org/wiki/Regularization_(mathematics) |
| Loss  | Loss functions  | no  | tmpurl_loss  | https://en.wikipedia.org/wiki/Loss_function |
| Calibration / calibrators  | calibrators used for producing probabilities  | no  | tmpurl_calib  | https://en.wikipedia.org/wiki/Calibration_(statistics)  |
| Learning rate  | what is learning rate in training  | no | tmpurl_lr  | There's no wikipage talking about learning rate. It's always mixed with SGD or the training algorithm using it. Deleting this since it's not an independent concept. |

**Comment by JRAlexander** [comment]
We can add to the glossary at any point.
